pkg/scheduler/util/utils.go

```diff
@@ -63,6 +66,32 @@ func GetPodPriority(pod *v1.Pod) int32 {
 	return scheduling.DefaultPriorityWhenNoDefaultClassExists
 }
 
+// GetEarliestPodStartTime returns the earliest start time of all pods that
+// have the highest priority among all victims.
+func GetEarliestPodStartTime(victims *api.Victims) *metav1.Time {
+	if len(victims.Pods) == 0 {
+		// should not reach here.
+		klog.Errorf("victims.Pods is empty. Should not reach here.")
+		return nil
+	}
+
+	earliestPodStartTime := victims.Pods[0].Status.StartTime
```

Comment:
Dereferencing pod.status.StartTime may not be safe. I think we need a helper function that gets pod StartTime and checks that pod.Status and pod.Status.StartTime are not nil. With that you shouldn't need to add pod status to all tests.